### PR TITLE
Docs: New Futo milestones

### PR DESCRIPTION
**Pull request by aviv926**
Because this is a change that is immediately going on I have linked it to release: 'V1.103.0',

image

### CODE REVIEW

File: docs/src/pages/milestones.tsx [hunk 1]
```diff
@@ -55,12 +55,22 @@ import {
...
 
 const items: Item[] = [
+  {
+    icon: mdiHandshakeOutline,
+    description: 'Joining Futo and Immich core team goes full-time',
+    title: 'Immich is joining FUTO!',
+    release: 'v1.103.0',
+    tag: 'v1.103.0',
+    date: new Date(2024, 4, 1),
+    dateType: DateType.RELEASE,
```
Comment (danieldietzler) [inline docs/src/pages/milestones.tsx:72]: Pretty sure you can also use `DateType.Date` or something and link it only to today?

Comment (aviv926) [reply docs/src/pages/milestones.tsx]: Yes you are right.
I removed the tag and added DateType.DATE,

Comment (danieldietzler) [reply docs/src/pages/milestones.tsx]: Can you also update the screenshot? :)

File: docs/src/pages/milestones.tsx [hunk 2]
```diff
@@ -55,12 +55,21 @@ import {
...
+  mdiHandshakeOutline,
 } from '@mdi/js';
 import Layout from '@theme/Layout';
 import React from 'react';
 import Timeline, { DateType, Item } from '../components/timeline';
 
 const items: Item[] = [
+  {
+    icon: mdiHandshakeOutline,
+    description: 'Joining Futo and Immich core team goes full-time',
```
Comment (jrasm91) [inline docs/src/pages/milestones.tsx:67]: Joined

**Review by jrasm91 — APPROVED** [review]

File: docs/src/pages/milestones.tsx (hunk 2 above)
Comment (zackpollard) [inline docs/src/pages/milestones.tsx:67]:
```suggestion
    description: 'Joined Futo and Immich core team goes full-time',
```

File: docs/src/pages/milestones.tsx [hunk 3]
```diff
@@ -55,12 +55,21 @@ import {
...
+    title: 'Immich is joining FUTO!',
```
Comment (zackpollard) [inline docs/src/pages/milestones.tsx:68]:
```suggestion
    title: 'Immich joins FUTO!',
```

**Comment by zackpollard** [comment]
I have made a PR #9212 to allow not specifying the release as well as the tag which will remove the top right element of the milestone.

**Review by zackpollard — APPROVED** [review]

File: docs/src/pages/milestones.tsx [hunk 4]
```diff
@@ -55,12 +55,20 @@ import {
...
 import Layout from '@theme/Layout';
 import React from 'react';
 import Timeline, { DateType, Item } from '../components/timeline';
 
 const items: Item[] = [
+  {
+    icon: mdiHandshakeOutline,
+    description: 'Joined Futo and Immich core team goes full-time',
+    title: 'Immich joins FUTO!',
+    date: new Date(2024, 4, 1),
```
Comment (Radu-C-Martin) [inline docs/src/pages/milestones.tsx:69]: 2024-04-01, or 2024-05-01?

Comment (zackpollard) [reply docs/src/pages/milestones.tsx:69]: In javascript, months are 0-indexed, so 4-1 is correct and as you can see in the screenshot shows the correct date.

Comment (Radu-C-Martin) [reply docs/src/pages/milestones.tsx:69]: ah, so silly of me. my bad 😅

**Pull request merged**